.github/workflows/run-unit-tests.yml

```diff
@@ -177,6 +177,8 @@ jobs:
             xcode-version: '~15.2'
```

Comment:
Give https://github.com/actions/runner-images/issues/10703 I think we can remove the `macos-13` runner